app/src/main/cpp/VRLayer.h

```diff
@@ -0,0 +1,149 @@
+/* -*- Mode: C++; tab-width: 20; indent-tabs-mode: nil; c-basic-offset: 2 -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef VR_LAYER_DOT_H
+#define VR_LAYER_DOT_H
+
+#include "vrb/gl.h"
+#include "vrb/MacroUtils.h"
+#include "vrb/Forward.h"
+
+#include "Device.h"
+
+#include <memory>
+#include <functional>
+#include <jni.h>
+
```

Comment:
Looks like functional and jni.h are left overs here to? I didn't see then being used any where.